src/osp/Active/SysAreaAssociate.cpp

```diff
@@ -33,16 +33,16 @@ using namespace osp;
 
 const std::string SysAreaAssociate::smc_name = "AreaAssociate";
 
-SysAreaAssociate::SysAreaAssociate(ActiveScene &rScene, Universe &uni) :
-       m_scene(rScene),
-       m_universe(uni),
-       m_updateScan(rScene.get_update_order(), "areascan", "physics", "",
-                    [this] { this->update_scan(); })
+SysAreaAssociate::SysAreaAssociate(ActiveScene &rScene, Universe &uni)
+ : m_scene(rScene)
+ , m_universe(uni)
+ , m_updateScan(rScene.get_update_order(), "areascan", "physics", "",
```

Comment:
This is weird, are you sure about this

Reply:
I've seen it around a few times, it's a pretty common style. Personally, I find it easier to work with. I never though of switching for consistency, but then jonesmz started using it in PR #19 so why not.